### PR TITLE
SV: Add chart dropdown

### PR DESCRIPTION
**Pull request by kalletlak**
# What? Why?
Fix https://github.com/cBioPortal/cbioportal/issues/4154

### CODE REVIEW

File: src/pages/studyView/styles.scss [hunk 1]
```diff
@@ -18,6 +18,38 @@
     .btn:active:focus {
         box-shadow: none; 
     }
+
+    .Select {
```
Comment (zhx828) [inline src/pages/studyView/styles.scss]: Doesn't the style come with the component already?

File: src/pages/studyView/StudyViewPageStore.ts [hunk 2]
```diff
@@ -306,26 +325,76 @@ export class StudyViewPageStore {
...
+            // TODO: this is a temorary solution util we figure out the chartVisibility logic
+            let osStatusFlag = false;
+            let osMonthsFlag = false;
+            let dfsStatusFlag = false;
+            let dfsMonthsFlag = false;
+            let mutationCountFlag = false;
+            let fractionGenomeAlteredFlag = false;
+
+            clinicalAttributes.forEach(obj => {
+                if (obj.clinicalAttributeId === 'OS_STATUS') {
```
Comment (zhx828) [inline src/pages/studyView/StudyViewPageStore.ts]: Let's make all these strings to const, similarly what we have for MUTATION_COUNT

File: src/pages/studyView/styles.scss (hunk 1 above)
Comment (kalletlak) [reply src/pages/studyView/styles.scss]: seems not

**Comment by kalletlak** [comment]
@zhx828 
1. I dont see any console errors
2. Cannot reproduce it
3. I don't see any error message
4. I don't think we can add loading gif, we may need to change in the library

**Comment by zhx828** [comment]
@jjgao @kalletlak should we add an option `Add the default charts` after clicking on the `Clear`?

**Comment by zhx828** [comment]
@kalletlak Another issue: the percentage of Mutated Genes/CNA Genes table are way high
screen shot 2018-08-30 at 10 45 06 am

Reproduce:
- Open study: http://www.cbioportal.org/newstudy?id=acc_tcga_pan_can_atlas_2018#summary
- Clear all charts
- Add Ethnicity Category from the dropdown
- Select the 8 slice
- Take a look the drop down

File: package.json [hunk 3]
```diff
@@ -139,7 +139,7 @@
     "mobx-react-devtools": "^4.2.11",
     "mobx-react-router": "^3.1.2",
     "mobx-utils": "^2.0.1",
-    "mobxpromise": "^1.3.1",
+    "mobxpromise": "github:cbioportal/mobxpromise#master",
```
Comment (zhx828) [inline package.json]: Why?

Comment (zhx828) [reply package.json]: Just saw the message in the channel

**Comment by zhx828** [comment]
Created another issue https://github.com/thehyve/react-select-checked/issues/23

File: src/pages/studyView/styles.scss (hunk 1 above)
Comment (zhx828) [reply src/pages/studyView/styles.scss]: The style needs to be updated, especially the line hight. Please see the style in coExpression folder under resultsView as example.

**Pull request merged**